lib/rubocop/target_ruby.rb

```diff
@@ -0,0 +1,153 @@
+# frozen_string_literal: true
+
+module RuboCop
+  # The kind of Ruby that code inspected by RuboCop is written in.
+  class TargetRuby
+    # 2.3 is the oldest officially supported Ruby version.
+    DEFAULT_VERSION = 2.3
```

Comment:
It seems this is only used in a single place, and we might be better off using `KNOWN_RUBIES.first` instead, to avoid having two sources of truth. WDYT?

Reply:
Good idea. At least we should have `DEFAULT_VERSION = KNOWN_RUBIES.first`. I'll see if it makes sense to get rid of the constant completely.

Reply:
I love this idea!

Reply:
I'll keep the constant for now. It is actually used in two places.